data/brands/amenity/bicycle_rental.json

```diff
@@ -4277,14 +4277,23 @@
       }
     },
     {
-      "displayName": "金門縣政府",
+      "displayName": "金門公共自行車",
       "id": "af5935-1bb859",
       "locationSet": {"include": ["tw"]},
-      "tags": {
-        "amenity": "bicycle_rental",
-        "brand": "金門縣政府",
-        "name": "金門縣政府",
-        "operator": "金門縣政府"
+      "matchNames": ["k bike"],
+      "tags": {
+        "amenity": "bicycle_rental",
+        "brand": "金門公共自行車",
+        "brand:en": "K-Bike",
+        "brand:zh": "金門公共自行車",
+        "brand:wikidata": "Q30942379",
+        "brand:wikipedia": "zh:金門縣公共自行車租賃系統",
+        "name": "金門公共自行車",
+        "name:en": "K-Bike",
+        "name:zh": "金門公共自行車",
+        "operator": "金門縣政府",
+        "operator:type": "government",
+        "website": "https://www.k-bike.com.tw/"
```

Comment:
We don't support website tags in the NSI - this data can be accessed through the linked Wikidata page
```suggestion
```